index.html

```diff
@@ -7,18 +7,10 @@
 style: index.css
 ---
 
-<!--Photos-->
-<div class="section">
-    <div class="home-photos">
-        <img src="imgs/Row.jpg">
-        <img src="imgs/BunnyEars.jpg">
-    </div>
-</div>
-
 <!--Details-->
-<div class="section">
-    <h2>Details</h2>
-    <div>
+<div class="top-container">
+    <div class="details-flex">
```

Comment:
good change, athough i would recommend using the 
```
<div class="details-container">
    <div class="details-item">
```
classes so you dont have to write extra CSS